README.md

```diff
@@ -1,24 +1,70 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Microverse School > Rails > #3 Members Only
 
```

Comment:
## Optional Changes ♻️ 
- [ ] Let's add a screenshot to our site's UI here, please ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)).